docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -8,92 +8,99 @@ ms.author: mikerou
     01. Upgrade NuGet packages to newer versions, if needed.
-    01. Use the [.NET Portability Analyzer][apiport] to understand .NET dependencies.
-01. Migrate the project file to the new SDK-style format.
-    01. Choose whether to target both .NET Core and .NET Framework or only .NET Core.
+    01. Use the [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md) to understand .NET dependencies.
+
+01. Migrate the project file to the new SDK-style format:
+
+    01. Choose whether to target both .NET Core and .NET Framework, or only .NET Core.
     01. Copy relevant project file properties and items to the new project file.
-01. Fix build issues.
-    01. Add a reference to the [Microsoft.Windows.Compatibility] package.
+
+01. Fix build issues:
+
+    01. Add a reference to the [Microsoft.Windows.Compatibility](https://www.nuget.org/packages/Microsoft.Windows.Compatibility/) package.
     01. Find and fix API-level differences.
     01. Remove app.config sections other than appSettings or connectionStrings.
     01. Regenerate generated code, if needed.
-01. Runtime testing.
+
+01. Runtime testing:
+
     01. Confirm the ported app works as expected.
-    01. Beware of NotSupportedExceptions.
+    01. Beware of <xref:System.NotSupportedException> exceptions.
 
 ## About the sample
 
-This guide references the [Bean Trader sample app][sample-github] because it uses a variety of dependencies similar to those real-world WPF apps might have. The app isn't large but is meant to be a step up from 'Hello World' in terms of complexity. The app will demonstrate some issues users may encounter while porting real applications. The app communicates with a WCF service so, in order for it to run properly, so you'll also need to run the BeanTraderServer project (available in the same GitHub repository) and make sure the BeanTraderClient's configuration points at the correct endpoint (by default, the sample assumes the server is running on the same machine at *http://localhost:8090*, which will be true if you just launch **BeanTraderServer** locally).
+This article references the [Bean Trader sample app](https://github.com/dotnet/windows-desktop/tree/master/Samples/BeanTrader) because it uses a variety of dependencies similar to those real-world WPF apps might have. The app isn't large but is meant to be a step up from 'Hello World' in terms of complexity. The app will demonstrate some issues users may encounter while porting real applications. The app communicates with a WCF service so, in order for it to run properly, so you'll also need to run the BeanTraderServer project (available in the same GitHub repository) and make sure the BeanTraderClient's configuration points at the correct endpoint (by default, the sample assumes the server is running on the same machine at `http://localhost:8090`, which will be true if you just launch **BeanTraderServer** locally).
```

Comment:
```suggestion
This article references the [Bean Trader sample app](https://github.com/dotnet/windows-desktop/tree/master/Samples/BeanTrader) because it uses a variety of dependencies similar to those real-world WPF apps might have. The app isn't large but is meant to be a step up from 'Hello World' in terms of complexity. The app will demonstrate some issues users may encounter while porting real applications. The app communicates with a WCF service so, in order for it to run properly, you'll also need to run the BeanTraderServer project (available in the same GitHub repository) and make sure the BeanTraderClient's configuration points at the correct endpoint (by default, the sample assumes the server is running on the same machine at `http://localhost:8090`, which will be true if you just launch **BeanTraderServer** locally).
```